seat.c

```diff
@@ -216,6 +221,25 @@ handle_new_pointer(struct cg_seat *seat, struct wlr_pointer *wlr_pointer)
 	map_input_device_to_output(seat, &wlr_pointer->base, wlr_pointer->output_name);
 }
 
+static void
+handle_virtual_pointer(struct wl_listener *listener, void *data)
+{
+	struct cg_seat *seat = wl_container_of(listener, seat, new_virtual_pointer);
+	struct wlr_virtual_pointer_v1_new_pointer_event *event = data;
+	struct wlr_virtual_pointer_v1 *pointer = event->new_pointer;
+	struct wlr_input_device *device = &pointer->pointer.base;
+
+
+	/* We'll want to map the device back to an output later, this is a bit
+	 * sub-optimal (we could just keep the suggested_output), but just copy
+	 * its name so we do like other devices
+	 */
+	device->name = strdup(event->suggested_output->name);
```

Comment:
Hm, I don't understand this line. `device->name` is the input device name, it's unrelated to the output name.

Did you mean `pointer->output_name`?